app/src/main/java/com/goldenraven/padawanwallet/ui/wallet/SendScreen.kt

```diff
@@ -227,15 +288,26 @@ private fun TransactionAmountInput(amount: MutableState<String>) {
             singleLine = true,
             textStyle = TextStyle(fontFamily = ShareTechMono, color = md_theme_dark_onBackground),
             label = {
-                Text(
-                    text = "Amount (satoshis)",
-                )
+                when (transactionOption) {
+                    TransactionType.SEND_ALL -> {
+                        Text(text = "Amount (Send All)")
```

Comment:
I think we should consider showing the amount here when the user chooses the "Send all" option. How about a quick `Wallet.getBalance()` to display the amount that is about to be sent?